app/views/payouts/list.njk

```diff
@@ -83,12 +82,40 @@ Payments to your bank account - GOV.UK Pay
       </tbody>
     </table>
   {% else %}
-    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-    {% if payoutsReleaseDate %}
-    <p class="govuk-body">No payments to your bank account found on or after {{ payoutsReleaseDate.format('D MMMM YYYY') }}.</p>
-    <p class="govuk-body">For historic payments to your bank account please <a class="govuk-link" href="https://www.payments.service.gov.uk/support/">contact support.</a></p>
+    {% if filterLiveAccounts %}
+      <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+      {% if payoutsReleaseDate %}
+        <p class="govuk-body">No payments to your bank account found on or after {{ payoutsReleaseDate.format('D MMMM YYYY') }}.</p>
+        <p class="govuk-body">For historic payments to your bank account please <a class="govuk-link" href="https://www.payments.service.gov.uk/support/">contact support.</a></p>
+      {% else %}
+        <p class="govuk-body">No payments to your bank account found.</p>
+      {% endif %}
     {% else %}
-    <p class="govuk-body">No payments to your bank account found.</p>
+      <h2 class="govuk-heading-m">Sample Report</h2>
+      <p class="govuk-body">This is a sample report. If you make test, payments, you'll be able to see a copy of the report using your test payment.</p>
+
+      <table class="govuk-table" id="payout-list">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Service</th>
+            <th scope="col" class="govuk-table__header govuk-table__header--numeric">Amount</th>
+            <th scope="col" class="govuk-table__header">View transactions</th>
+          </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+          <tr class="govuk-table__row">
+            <td scope="row" class="govuk-table__cell">Test Service</td>
+            <td class="govuk-table__cell govuk-table__cell--numeric">
+              {{ 10000 | penceToPoundsWithCurrency }}
+            </td>
+            <td class="govuk-table__cell">
+              <a class="govuk-link govuk-link--no-visited-state" href="public/csv/sample_reporting_payout.csv">
```

Comment:
Does this link work? I tried testing locally and got page not found. Does it also need to have the `download` attribute?

Reply:
Looks like the href should be `/csv/sample_reporting_payout.csv`

Reply:
Nice spot will change that now!